tests/consensus_tests/test_partial_snapshot.py

```
@@ -186,3 +242,9 @@ def scroll_points(peer_url: str):
 
 def upsert(peer_url: str, points: int, offset = 0):
     upsert_random_points(peer_url, points, offset = offset, batch_size = 10, with_sparse_vector = False)
+
+def delete(peer_url: str, until_id: int, from_id = 0):
+    resp = requests.post(f"{peer_url}/collections/{COLLECTION}/points/delete?wait=true", json = {
+        "points": [id for id in range(from_id, until_id)],
+    })
+    assert_http_ok(resp)
```

Ruff (0.11.9)

`assert_http_ok` may be undefined, or defined from star imports

(F405)